server_admin/topics/authentication/webauthn.adoc

```diff
@@ -167,27 +166,61 @@ A new user carries out the following operations :
+
+Administrator may typically require that Security Keys registered by users for the WebAuthn passwordless authentication must meet different
+(usually stronger) requirements. For example, those security keys may require users to authenticate to that security key using a PIN, or the
+security key should be attested with stronger certificate authority.
+
+Because of this situation, {project_name} allows administrator to configure separate `WebAuthn Passwordless Policy`. There is a separate required action
+of type `Webauthn Register Passwordless` and separate authenticator of type `WebAuthn Passwordless Authenticator`.
+
+===== Setup
+
+The setup procedure of WebAuthn passwordless support is the following :
+
+* Register new required action for WebAuthn passwordless support. Use the same steps as described <<_webauthn-register, above>>
+with the only difference, that you need to register the action called `Webauthn Register Passwordless`.
+
+* Configure the policy. You can use same steps and configuration options as described <<_webauthn-policy, above>>, however you
+need to configure them in the admin console in the tab `WebAuthn Passwordless Policy`. You can configure this policy as you want, however
+typically the requirements for the security key will be stronger than for the two-factor policy. For example the `User Verification Requirement` can
+be set to `Required` when you configure the passwordless policy.
+
+* Finally configure the authentication flow. Let's assume that we will use same flow called `WebAuthn Browser` as described
+<<_webauthn-authenticator-setup, above>>, but
+we will configure it as follows:
+
+** The `WebAuthn Browser Forms` subflow will contain `Username Form` as the first authenticator. This setting means that the user
+will provide just his or her username as the first step.
+
+** There will be a required subflow, which can be named for example `Passwordless Or Two-factor` . This setting indicates that user can
+authenticate either with Passwordless WebAuthn credential or with Two-factor authentication.
 
-A user carries out the following operations on the <<_account-service, `User Account Service`>>  :
+** Flow will contain `WebAuthn Passwordless Authenticator` as first alternative.
 
-- View the `Account` page.
+** The second alternative will be a subflow named for example `Password And Two-factor Webauthn`. This subflow will contain `Password Form` and
+`WebAuthn Authenticator`.
 
-===== Edit Registered WebAuthn Authenticator
+The full configuration of the flow will look like this:
 
-A user can edit the following information :
+image:images/webauthn-passwordless-flow.png[]
 
-- Label (WebAuthn authenticator's label the user entered on registering it)
+You can now add `WebAuthn Register Passwordless` as the required action to some user to test this. During the first authentication, the user will
```

Comment:
And could we have description / image of sole
_==== Passwordless WebAuthn_ configuration? (the section listed above is describing the combination WebAuthn Passwordless together with WebAuthn Two-Factor AFAICT)

Would the proper flow configuration if wanting only `WebAuthn Passwordless` look like this;?

![webauth_passwordless](https://user-images.githubusercontent.com/8414918/72804762-7a4b6380-3c51-11ea-80e9-d06a80c4a0b6.png)

WDYT?

Reply:
Yes, but I think that `WebAuthn Passwordless` will be usually used together with two-factor as an alternative and not sure if there is much point of adding another configuration example. It should be clear enough how it works from the previous example and too much examples makes the docs more complex (and refactoring harder etc) :) So my preference would be to not introduce more example. Are you fine with that?

Reply:
Sure, np with that. Since `WebAuthn Passwordless` is already partially covered in the previous example, should be possible the user to follow the example from the `WebAuthn Passwordless together with Two-Factor` example.